lessons/pydata/databases/index.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>Tady je možné použít <code>%pip</code>  nebo <code>!python -m pip</code>  — minimálně to první už někde v předchozích lekcích použito je.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Je ok to použít v "obecném" notebooku? Já třeba nejedu přes venv a nechci tento příkaz spouštět při "run all" :) A nebo by to mohlo být zakomentované...</p>

Reply:
<p>Klidně zakomentovat nebo jen zmínit v textu a kdo to bude dělat, tak si jej nakopíruje do buňky nebo terminálu.</p>